### PR TITLE
[CB-110] Fix displaying language in list enrolled students' profile information

### PR DESCRIPTION
**Pull request by Masergik**
**Description:** Added ability to take `Preferred language` in `enrolled_students_features` function for displaying language in the "list enrolled students' profile information" under instructor.
Student can set up `Preferred language` in his profile or account page.

**Youtrack:** https://youtrack.raccoongang.com/issue/CB-110

### CODE REVIEW

**Comment by raccoondeploy** [comment]
Refer to this link for build results (access rights to CI server needed): 

<a href='http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/17/'>http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/17/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/259/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/259/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/260/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/260/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/261/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/261/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/262/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/262/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/263/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/263/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/264/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/264/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/265/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/265/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/266/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/266/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/267/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/267/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/268/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/268/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/269/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/269/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/270/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/270/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/271/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/271/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/272/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/272/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/273/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/273/</a>

File: lms/djangoapps/instructor_analytics/basic.py [hunk 1]
```diff
@@ -293,6 +293,9 @@ def extract_student(student, features):
             if include_enrollment_mode:
                 student_dict['enrollment_mode'] = enrollment_mode
 
+        preferred_lang = list(profile.language_proficiencies.values('code'))
```
Comment (idegtiarov) [inline lms/djangoapps/instructor_analytics/basic.py]: `values_list('code', flat=True)` returns a list-like object, so it won't be needed to cast to the list and take value by the key `code`

Comment (Masergik) [reply lms/djangoapps/instructor_analytics/basic.py]: Done

**Comment by raccoondeploy** [comment]
Refer to this link for build results (access rights to CI server needed): 

<a href='http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/18/'>http://jenkins-ci.raccoongang.com/job/hawthorn-python-unittests-pr/18/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/274/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/274/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/275/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/275/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/276/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/276/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/277/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/277/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/278/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/278/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/279/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/279/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/280/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/280/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/281/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/281/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/282/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/282/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/283/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/283/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/284/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/284/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/285/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/285/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/286/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/286/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/287/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/287/</a>
<a href='http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/288/'>http://jenkins-ci.raccoongang.com/job/edx-platform-test-subset/288/</a>

File: lms/djangoapps/instructor_analytics/basic.py [hunk 2]
```diff
@@ -293,6 +293,9 @@ def extract_student(student, features):
             if include_enrollment_mode:
                 student_dict['enrollment_mode'] = enrollment_mode
 
+        preferred_lang = profile.language_proficiencies.values_list('code', flat=True)
+        student_dict['language'] = preferred_lang[0] if preferred_lang else profile.language
```
Comment (arsentur) [inline lms/djangoapps/instructor_analytics/basic.py]: Are you sure that `preferred_lang[0]` return current(latest) value? Maybe we need to use last element of the list?

Comment (Masergik) [reply lms/djangoapps/instructor_analytics/basic.py]: Student can chose only one language in profile or account page

**Pull request merged**